templates/default/goiardi.conf.erb

```diff
@@ -21,8 +21,10 @@ log-level = "<%= @log_level %>"
 
 # If these two options are both set, goiardi will save the data store and index
 # data to disk. It will cause an error to only set one of the options
+<% unless @use_mysql or @use_postgresql %>
 index-file = "<%= @rundir %>/goiardi-index.bin"
```

Comment:
One note here: the index file should still be set, even when using mysql or postgres for the data store.